README.md

```diff
@@ -14,7 +14,7 @@ While Swift Distributed Tracing allows building all kinds of _instruments_, whic
 
 This project uses the context progagation type defined independently in:
 
-- 🧳 [swift-distributed-tracing-baggage](https://github.com/apple/swift-distributed-tracing-baggage) -- [`Baggage`](https://apple.github.io/swift-distributed-tracing-baggage/docs/current/InstrumentationBaggage/Structs/Baggage.html) (zero dependencies)
+- 🧳 [swift-service-context](https://github.com/apple/swift-service-context) -- [`Baggage`](https://apple.github.io/swift-service-context/docs/current/ServiceContextModule/Structs/Baggage.html) (zero dependencies)
```

Comment:
Should these links point to the Swift Package Index hosted docs instead? This one gives me a 404.

Reply:
This one should be 404 because `Baggage` is renamed. +1 for switching to SPI though.